### PR TITLE
feat(utilities): import getFirstFocusable method from Fabric

### PR DESCRIPTION
**Pull request by eoh**
Importing `getFirstFocusable` method from Fabric, to be used with `SkipLinks` component in MC

### CODE REVIEW

File: src/util/getFirstFocusable [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { getFirstFocusable } from '@uifabric/utilities/lib/focus';
+
+export { getFirstFocusable };
```
Comment (unindented) [inline src/util/getFirstFocusable]: This can be condensed into one line:

```ts
export { getFirstFocusable } from '@uifabric/utilities/lib/focus';
```

**Comment by unindented** [comment]
Oh shit, I didn't notice. That's what we get for not writing tests. @eoh can you create a new PR that adds the extension and a unit test?

**Comment by BrendanMcK** [comment]
Is there a meaningful test that can be written here, though? Or is it the case that the import itself *is* the test, with an empty describe? Because we're exporting directly from fabric, it's not like we even have an intermediate wrapper class to exercise.

**Comment by unindented** [comment]
Maybe just importing it and expecting that the thing you got is a function?

**Pull request merged**